lib/routes/account.js

```diff
@@ -646,6 +734,7 @@ module.exports = function (
             response.verificationMethod = 'email'
             response.verificationReason = 'login'
           }
+          recordSecurityEvent() // don't block response recording db event
```

Comment:
Ok, so if a user signs in from the accounts.firefox.com page, instead of trying to enable Sync, we'll record an `account.login` event, tied to that session. That session will never verified, so we'll just never view the event as verified. If that changes in the future, the events will be upgraded automatically.